prow/cmd/hook/main.go

```diff
@@ -201,6 +216,16 @@ func main() {
 		slackClient = slack.NewFakeClient()
 	}
 
+	var rocketChatClient *rocketchat.Client
+	if !o.dryRun && string(secret.GetSecret(o.rocketChatWebhookFile)) != "" {
+		logrus.Info("Using real rocketchat client.")
+		rocketChatClient = rocketchat.NewClient(secret.GetTokenGenerator(o.rocketChatWebhookFile))
+	}
+	if rocketChatClient == nil {
+		logrus.Info("Using fake rocketchat client.")
```

Comment:
Does not seem like something we want to log if we do not want to use RocketChat at all

Reply:
I agree, but it is the same for slack.
Should I remove it for rocketchat?
Or for both?

Reply:
Let's keep it for both, no big deal